src/Instances/AttributeChange.lua

```diff
@@ -0,0 +1,36 @@
+--!strict
+
+--[[
+	A special key for property tables, which allows users to connect to
+    an attribute change on an instance.
+]]
+
+local Package = script.Parent.Parent
+local PubTypes = require(Package.PubTypes)
+local logError = require(Package.Logging.logError)
+local xtypeof = require(Package.Utility.xtypeof)
+
+local function AttributeChanged(attributeName: string): PubTypes.SpecialKey
+	local attributeKey = {}
+	attributeKey.type = "SpecialKey"
+	attributeKey.kind = "AttributeChange"
+	attributeKey.stage = "observer"
+
+    function attributeKey:apply(callback: any, applyTo: Instance, cleanupTasks: {PubTypes.Task})
```

Comment:
There are mixed tabs/spaces for indenting here. Line 17 is a tab, line 19 is four spaces.

Reply:
Fixed :D